### ISSUE TITLE
Extend flake8 line length to 100 characters

### ISSUE THREAD

**Issue by m1yag1** [openstax/os-automation#13]
We're not using teletypes anymore.

Actually, the big reason for this change is that locators tend to be a little long. this allows us to keep them on one line.

**Comment by gregfitch** [comment]
I would counter that it's not for teletypes at this point...from a previous Google person:

"At Google, our Python style guide dictates 80 characters. Java is the only exception at 100. Even 100 can be awkward to fit into. Lord knows I've renamed variables to avoid splitting a line on several occasions.

But what a lot of posters here seem to miss is that it isn't about the width of screens for editing, it's about the width of screens for reviewing so you need twice the real estate.

Add in doing interactive merges and you need three times the width."

**Comment by m1yag1** [comment]
Google also does two character indents which was done to get more characters on a line (rather than extending). Changing indent level to get two characters is just crazy to me. Raymond Hettinger calls these "minor atrocities". I've been a strict observer of the 79 char limit for a long time and I seldom make exceptions. I believe from experience working with these locators and general verboseness of test code that 100 chars is a good exception to make. The decision wasn't made lightly and was made keeping Raymond Hettinger's advice in mind in this video https://youtu.be/wf-BqAjZb8M?t=4m36s.

Basically, any absolute number is dangerous.
